### PR TITLE
Makes services the initial topology if available

### LINKED ISSUE #1809: make 'pods' the initial view, when available
@jml pointed out that when running k8s, the container view is rarely what one wants to look at. So perhaps we should make the 'pods' view the default initial view, if available.

### PR DESCRIPTION
**Pull request by foot**
- Otherwise reverts to containers

Fixes #1809

### CODE REVIEW

**Comment by rade** [comment]
> Otherwise reverts to containers

The prioritisaion should be services -> deployments -> replica sets  -> pods -> containers. Says @tomwilkie.

**Comment by foot** [comment]
Cool, will update

File: client/app/scripts/utils/topology-utils.js [hunk 1]
```diff
@@ -1,5 +1,31 @@
 import _ from 'lodash';
-import { is as isDeepEqual, Map as makeMap, Set as makeSet } from 'immutable';
+import { is as isDeepEqual, Map as makeMap, Set as makeSet, List as makeList } from 'immutable';
+
+
+//
+// top priority first
+//
+const TOPOLOGY_DISPLAY_PRIORITY = [
```
Comment (davkal) [inline client/app/scripts/utils/topology-utils.js]: I'd prefer it if this was backend-driven via a priority field.

**Comment by davkal** [comment]
Code looks good.

Implementation-nit: priority should be driven by backend, so that when new topos are added, UI does not have to be involved.

UX-nit: If Services is the most important one, it should be at the top. Any introduction of priority ordering trumps representation model ordering (lower topos are groupings of higher topos). This needs to be changed in the backend.

**Comment by davkal** [comment]
> priority should be driven by backend, so that when new topos are added, UI does not have to be involved.
Actually, we should put this up for discussion in the next meeting.

**Comment by davkal** [comment]
Can't we make service the main topo?

screen shot 2016-08-29 at 16 31 26

**Comment by davkal** [comment]
If we cant completely flip the order, I'd say let's stick to the original one.

**Comment by foot** [comment]
Node type, followed by its aggregates in order of most interest breaks
strict "spatial alignment" w/ the priorities we've set up but I think it
still works. If no aggs fall back to basic type.

**Comment by davkal** [comment]
I mostly have a problem with a sub-menu entry being the most important one (decided by priority). I feel that it makes the menu harder to grok. 
An alternative would be dynamic prio ordering in the backend, so that the default one is the top one.

**Pull request merged**